spec/app/domain/authentication/authn_k8s/inject_client_cert_spec.rb

```diff
@@ -182,6 +187,7 @@
           expect { injector.(conjur_account: account,
                             service_id: service_id,
                             csr: csr,
+                            cert_installation_path: nil_cert_installation_path,
```

Comment:
I'd just get rid of `nil_cert_installation_path` and use the literal `nil`.  Seeing:
```
cert_installation_path: nil
```
will be clearer.

Reply:
👍